### DISCUSSION TITLE
Dpdata installation problem

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#780] [Q&A]
Deepmd version is 1.3.3. When installing dpdata-0.2.1, we encounter the following problems. How can we solve them

![image](https://user-images.githubusercontent.com/73742098/122733831-2282b300-d2b0-11eb-895f-6eb9b1cd1d6b.png)

**Comment by Ericwang6** [comment]
In the source code directory of dpdata, just run `pip install .` or `python setup.py install`

**Reply by felix5572** [reply]
Maybe the connection problem. try  install yourself? `git clone` dpdata and `pip install .` or `python setup.py install` ?
